packages/fxa-auth-server/test/scripts/remove-accounts.js

```diff
@@ -0,0 +1,190 @@
+'use strict';
+
+import {
+  retreiveUnverifiedAccounts,
+  cancelSubscriptionsAndDeleteCustomer,
+  issueRefund,
+} from '../../scripts/remove-accounts';
```

Comment:
Q: SHould this be `import` or `require()`? We seem to use `require()` below on L10-15.

Reply:
yes, done